portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,243 @@
 <!DOCTYPE html>
 <html>
-  <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet"/>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+        <div class="container">                       
+            <img class="profilePhoto" src="../images/profilePhoto.jpg" alt="profile image" width=130 height=130 />
+            <div>
+                <h1 class="name">Tanya Chauhan</h1>
+                <div>
+                    <a href="https://www.facebook.com/tanya.chauhan.09">
+                        <img class="links" src="../images/facebookIcon.svg" width=30 height=30>
+                    </a>
+                    <a href="https://github.com/TanyaChauhan9">
+                        <img class="links" src="../images/githubIcon.svg" width=30 height=30>
+                    </a>
+                    <a href="https://www.linkedin.com/in/tanya3/">
+                        <img class="links" src="../images/linkedin.svg" width=30 height=30>
+                    </a>
+                    <a href="https://twitter.com/tanyachauhan99">
+                        <img class="links" src="../images/twitterIcon.png" width=30 height=30>
+                    </a>
+                </div>
+            </div><!--//profile-->              
+        </div><!--//container-->
+        <div class="content">
+            <div class="leftSide">
+                <div class="Aboutme">
+                    <h2>About me</h2>
+                    <p>Hi! I'm Tanya. 
+                    I'm pursuing my Bachelor's in Mathematics and Computing from Indian Institure of Techology, Guwahati. 
+                    Things that interest me the most are music, coding, working out, novels, astrophysics and a starry night sky! 
```

Comment:
Starry night sky..... I could get a glimpse of stars at night time when I was in Guwahati but in Hyderabad I rarely find them. I guess the same would be the case for most of the metros.

Reply:
Yes. Same for Delhi as well :/